spec/unit/recipes/default_spec.rb

```diff
@@ -3,12 +3,31 @@
 describe 'osl-ceph::default' do
   ALL_PLATFORMS.each do |p|
     context "#{p[:platform]} #{p[:version]}" do
-      cached(:chef_run) do
-        ChefSpec::SoloRunner.new(p).converge(described_recipe)
-      end
+      include_context 'chef_server', p
+      include_context 'common_stubs'
```

Comment:
I liked having the separation in case I needed something different down the road.

Reply:
What do you mean by different?

If you are only adding stubs then this separation isn't necessary, you could just append them after the `chef_server` context. Removing unused stubs isn't required, chefspec will still run fine with unnecessary stubs.

Even if you don't use `chef_server`, you can include `common_stubs` separately.

What I'm visualizing is a context within a context, where `common_stubs` is included in `chef_server`. This means that the `common_stubs` context is still 'separate' in the sense that it can be called without `chef_server` if you don't want the default context.

Reply:
Oh I see what you're saying. It looks like you can do that. Let me adjust it so that it includes it.